app/qml/main.qml

```diff
@@ -292,6 +292,7 @@ ApplicationWindow {
         nameFilters: ["All files (*)", "PNG files (*.png)", "BMP files (*.bmp)", "SLP files (*.slp)", "STP files (*.stp)"]
         defaultSuffix: projectManager.projectExtensionForType(Project.ImageType)
         onAccepted: loadProject(file)
+        folder: (project && project.dirUrl) ? project.dirUrl : ""
```

Comment:
It seems that this only works for the first project - if you open another project in a different directory, the dialog doesn't open in that directory on subsequent opens. Do you see that behaviour too? Which OS are you on?

I thought https://codereview.qt-project.org/c/qt/qtquickcontrols2/+/250538 might have fixed this, but that also only accounts for the first time. I wonder if it's intentional... I created https://bugreports.qt.io/browse/QTBUG-76292 just in case.

Reply:
I test it for windows 10 and It works fine. I will test it for Arch Linux today too.

Reply:
OK, well I guess it doesn't hurt to have it here anyway. However, you can remove the project.dirUrl check, because I don't think it's necessary. Instead, it should look like this:

    folder: project && project.loaded ? project.dirUrl : ""